Update alias.md (#5597)
segmentio/segment-docs: src/connections/spec/alias.md

The `alias` method is used to merge two user identities, effectively connecting two sets of user data as one. This is an advanced method, but it is required to manage user identities successfully in some of our destinations.

> info "Alias and Unify"

> > Alias call cannot be used to merge profiles in [Unify](https://segment.com/docs/unify/). For more information on how Unify merges user profiles, please check out our documentation on [Identity Resolution](https://segment.com/docs/unify/identity-resolution/). 

Reviewer: ```suggestion
> The alias call can't be used to merge profiles in [Unify](/docs/unify/). For more information on how Unify merges user profiles, see the documentation on [Identity Resolution](/docs/unify/identity-resolution/). 
```
